common/src/main/java/com/microsoft/identity/common/internal/cache/MsalOAuth2TokenCache.java

```diff
@@ -862,7 +862,7 @@ private void setToCacheRecord(@NonNull final CacheRecord target,
     @Override
     public List<AccountRecord> getAccounts(@Nullable final String environment,
                                            @NonNull final String clientId) {
-        final String methodName = ":getAccounts";
+        final String methodName = ":getAllAccounts";
```

Comment:
is this intentional?

Reply:
good catch, Android Studio auto-rename might have refactored this, fixed